Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -54,9 +54,16 @@ def PyInit(self):
                              direction=Direction.Output),
                              ("Name the workspace that will contain the " +
                               "calibration coefficients"))
+        self.declareProperty("Temperature",
+                             defaultValue=Property.EMPTY_DBL,
+                             validator=FloatBoundedValidator(lower=0.0),
+                             direction=Direction.Input,
+                             doc=("Temperature during the experiment."))
         return
 
     def validateInputs(self):
+        """ Validate the inputs
+        """
         issues = dict()
         inws = self.getProperty("VanadiumWorkspace").value
         run = inws.getRun()
```

Comment:
This line needs some protection, if you have a table workspace selected in the workspace list when you bring up the algorithm dialog box, then inws will be None, and will crash Mantid with a 'NoneType' object has no attribute 'getRun'.

This happened to me several times while testing.